src/ir_Mitsubishi.cpp

```diff
@@ -870,18 +853,9 @@ void IRMitsubishi136::stateReset(void) {
   // The state of the IR remote in IR code form.
   // Known good state obtained from:
   //   https://docs.google.com/spreadsheets/d/1f8EGfIbBUo2B-CzUFdrgKQprWakoYNKM80IKZN4KXQE/edit#gid=312397579&range=A10
-  remote_state[0] = 0x23;
-  remote_state[1] = 0xCB;
-  remote_state[2] = 0x26;
-  remote_state[3] = 0x21;
-  remote_state[4] = 0x00;
-  remote_state[5] = 0x40;
-  remote_state[6] = 0xC2;
-  remote_state[7] = 0xC7;
-  remote_state[8] = 0x04;
-  remote_state[9] = 0x00;
-  remote_state[10] = 0x00;
-  checksum();  // Calculate the checksum which covers the rest of the state.
+  static const uint8_t kReset[kMitsubishi136StateLength] = {
+      0x23, 0xCB, 0x26, 0x21, 0x00, 0x40, 0xC2, 0xC7, 0x04};
+  memcpy(remote_state, kReset, kMitsubishi136StateLength);
```

Comment:
Only setting 9 of 11 items?

Reply:
Yep, actually it's 9 of 17. I'm fine with the rest of the array being zeros.
`const uint16_t kMitsubishi136StateLength = 17;`

See https://en.cppreference.com/w/c/language/array_initialization for how array initialisation works when it's shorter than the declared length.
i.e.
`int y[5] = {1,2,3}; // y has type int[5] and holds 1,2,3,0,0`

Reply:
Oh, and thanks for reviewing the code. ;-)

Reply:
Good, reason for noticing was other trailing 0x00, but seems it's only in ir_Neoclima.cpp@81